### PR TITLE
Bugfix cass stress gen minor fixes rebased

### PR DESCRIPTION
**Pull request by dahankzter**
This is a rebased version of the cassandra bugfix PR #38 

It also updates the monitoring dashboards to scylla 4.0.

### CODE REVIEW

File: examples/generic/grafana/values.yaml [hunk 1]
```diff
@@ -303,7 +314,22 @@ dashboardProviders: {}
 ##
 ## dashboards per provider, use provider name as key.
 ##
-dashboards: {}
+dashboards:
+  scylla:
+    scylla-cpu-3.0:
```
Comment (mmatczuk) [inline examples/generic/grafana/values.yaml:319]: Can we use 4.0

File: hack/cass-stress-gen.py [hunk 2]
```diff
@@ -17,28 +17,36 @@
...
+                    app: cassandra-stress
        
 """
 
 def parse():
     parser = argparse.ArgumentParser(description='Generate cassandra-stress job templates for Kubernetes.')
     parser.add_argument('--num-jobs', type=int, default=1, help='number of Kubernetes jobs to generate - defaults to 1', dest='num_jobs')
     parser.add_argument('--name', default='cassandra-stress', help='name of the generated yaml file - defaults to cassandra-stress')
-    parser.add_argument('--scylla-version', default='2.3.1', help='version of scylla server to use for cassandra-stress - defaults to 2.3.1', dest='scylla_version')
+    parser.add_argument('--scylla-version', default='3.0.10', help='version of scylla server to use for cassandra-stress - defaults to 2.3.1', dest='scylla_version')
```
Comment (mmatczuk) [inline hack/cass-stress-gen.py:49]: ?

Comment (mmatczuk) [reply hack/cass-stress-gen.py:49]: ?

File: hack/cass-stress-gen.py [hunk 3]
```diff
@@ -79,4 +87,4 @@ def create_job_list(args):
     else:
       f = open(args.name + '.yaml', 'w')
       f.write('\n---\n'.join(manifests))
-      f.close
\ No newline at end of file
+      f.close
```
Comment (mmatczuk) [inline hack/cass-stress-gen.py:90]: ?

File: hack/cass-stress-gen.py (hunk 2 above)
Comment (dahankzter) [reply hack/cass-stress-gen.py:49]: Just needs a bump. I have a PR that touches this script in progress. I can do it there.

Comment (dahankzter) [reply hack/cass-stress-gen.py:49]: Bumoing the default. I guess 4.0.0 or something would be better.

File: examples/generic/grafana/values.yaml (hunk 1 above)
Comment (dahankzter) [reply examples/generic/grafana/values.yaml:319]: Yes let's go for 4.0 across the board.

File: hack/cass-stress-gen.py (hunk 3 above)
Comment (dahankzter) [reply hack/cass-stress-gen.py:90]: I have no idea... Perhaps `f.close()` is what is meant although python can be truly magic sometimes.

**Pull request merged**